packages/components/src/color-palette/test/index.js

```diff
@@ -71,7 +71,7 @@ describe( 'ColorPalette', () => {
 			const isOpen = true;
 			const onToggle = jest.fn();
 
-			const renderedToggleButton = shallow( dropdown.props().renderToggle( { isOpen, onToggle } ).props.children );
+			const renderedToggleButton = shallow( dropdown.props().renderToggle( { isOpen, onToggle } ) );
```

Comment:
@gziolo would you mind taking a quick look at this change? I just want to make sure I fixed those tests correctly. 

After removing the tooltip wrapper in f493620, I was getting a failed test that said `ShallowWrapper can only wrap valid elements` on this line. I tried removing the `.props.children` bit at the end here (since the wrapper was removed, there are no child elements in this dropdown anymore), and that got the tests to pass. Is that the right move here? 

Thanks!

Reply:
Before it was a Tooltip component with a button and span. Now it's simply `Button` component so you no longer need to access `button` through children. Nicely done 👍

Reply:
⭐️ Thank you!